juriscraper/opinions/united_states/state/vt.py

```diff
@@ -45,7 +64,71 @@ def _process_html(self):
                 {
                     "url": name_url_span.xpath(".//a/@href")[0],
                     "name": name_url_span.text_content(),
```

Comment:
Not all party names start with capital letters (for example here: https://www.vermontjudiciary.org/opinions-decisions?facet_from_date=&facet_to_date=&f%5B0%5D=court_division_opinions_library_%3A1), you could use titlecase function here
```suggestion
                    "name": titlecase(name_url_span.text_content()),
```

Reply:
How annoying! I'll just add that you guys probably know this, but we have our own titlecase utility that's optimized for names (and thus better than Python's built-on one).

Reply:
Yes, I was referring to the juriscraper function in juriscraper/lib/string_utils.py, are you referring to this one? or do we have another?

Reply:
YEP! Luckily, we just have the one!

Reply:
Good catch, I updated the PR. But now tests are failing across the repo